talk.py

```diff
@@ -407,7 +407,7 @@ async def verify(ctx, user: typing.Optional[discord.Member]):
         descriptors.append("Interest is piling up for your debts.")
     if "based" in [y.name.lower() for y in user.roles]:
         descriptors.append("Thank you for boosting this server and making the bread game possible!")
-    if "this man got ten thousand bricks" in [y.name.lower() for y in user.roles]:
+    if "(Addicted to Bricks)²" in [y.name.lower() for y in user.roles]:
```

Comment:
As the role name is lowered, I guess you should either change capitalized letters or remove the `.lower()`

Reply:
Oop, you're 100% right there, my bad.
Nice catch!